jest.config.json

```diff
@@ -1,5 +1,6 @@
 {
   "verbose": true,
+  "silent": true,
```

Comment:
Do `verbose` and `silent` affect different output? Seems like they would be opposites.

Reply:
`silent` suppresses console output from the code under test
`verbose` enables detailed test pass/fail reporting output